packages/vite/src/node/plugins/css.ts

```diff
@@ -439,12 +439,61 @@ export function cssPlugin(config: ResolvedConfig): Plugin {
   }
 }
 
+const createStyleContentMap = () => {
+  const contents = new Map<string, string>() // css id -> css content
+  const scopedIds = new Set<string>() // whether that id of css is scoped
```

Comment:
The comment here is a bit confusing to mean as "whether" is usually used for describing booleans. Would something like this be more accurate?
```suggestion
  const scopedIds = new Set<string>() //  ids of css that are scoped
```